.github/workflows/build-and-push-docker-image.yml

```diff
@@ -29,7 +29,7 @@ jobs:
         uses: bahmutov/npm-install@v1
 
       - name: Compile TypeScript and build Docker images
-        run: yarn build-docker-ts
+        run: pnpm build-docker-ts
```

Comment:
this whole workflow needs more changes for pnpm

Reply:
Changed the workflow to use pnpm
[d9dc313](https://github.com/upleveled/preflight/pull/379/commits/d9dc313a2cb7967a7968f8f11391188e3f37d160)

Reply:
we usually have the pnpm/action-setup before actions/setup-node

Reply:
I copied the workflow from a different file and didn't think about what I was copying and if this was necessary. Made the changes in the workflow file in this commit
[d5bdc40](https://github.com/upleveled/preflight/pull/379/commits/d5bdc40ce542df8e6fb72be080689104be5f078d)